docs/src/main/docs/guides/91_mp-tutorial.adoc

```diff
@@ -0,0 +1,1267 @@
+ beans in the application.
+
+Create a `beans.xml` in the `src/main/resources/META-INF` directory
+ with the following content:
+
+[source,xml]
+.beans.xml file
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
+                           http://xmlns.jcp.org/xml/ns/javaee/beans_2_0.xsd"
+       version="2.0"
+       bean-discovery-mode="annotated"> <1>
+</beans>
+----
+
+<1> The `bean-discovery-mode` tells Helidon to look for the annotations
+ to discover Java beans in the application.
+
+== Update the POM to build the application
+
+Helidon MP applications are packaged into a JAR file and the dependencies
+ are copied into a `libs` directory.  Some more detailed explanation of 
+ why this particular packaging approach is recommended is provided on the 
+ <<guides/04_dockerfile,Creating Docker Images>> page. 
+ Update the Maven POM file to create the correct packaging.  Here is 
+ the updated file:
+
+[source,xml]
+.Updated POM file
+----
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.helidon.examples</groupId>
+    <artifactId>helidon-mp-tutorial</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+    <name>${project.artifactId}</name>
+
+    <properties>
+        <helidon.version>1.2.0</helidon.version>
+        <mainClass>io.helidon.examples.Main</mainClass>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>${maven.compiler.source}</maven.compiler.target>
+        <libs.classpath.prefix>libs</libs.classpath.prefix> <1>
+        <copied.libs.dir>${project.build.directory}/${libs.classpath.prefix}</copied.libs.dir>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.helidon</groupId>
+                <artifactId>helidon-bom</artifactId>
+                <version>${helidon.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.helidon.microprofile.bundles</groupId>
+            <artifactId>helidon-microprofile-2.2</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId> <2>
+                    <artifactId>maven-jar-plugin</artifactId>
+                    <version>2.5</version>
```

Comment:
For whatever it's worth this is an ancient version of this plugin.  As of this writing the [current version is 3.1.2](https://maven.apache.org/plugins/maven-jar-plugin/).

Reply:
that's the version used in the helidon archetype, i just copied. i can update it if you want, but might be a good idea to update the archetype too